### ISSUE TITLE
Change the order of labels

### ISSUE THREAD

**Issue by tetelevm** [ultralytics/yolov5#3841]
(It's funny that the `question` has a red emoji and a purple tag, and the `bug` has a purple emoji and a red tag)

To display the labels on a photo I use the code
```
    def get_predict(self): 
        predict = self.model(image)
        return predict.render()[0]
```
And there is a problem, which can be seen in the image

![img](https://user-images.githubusercontent.com/46541662/123957947-fb0ca400-d99b-11eb-8263-9138977d65b9.png)

As I understand it, rendering goes on probability reduction, and because of this objects with lower probability can block objects with higher probability. How to make the drawing of labels in reverse, from less to more?

**Comment by glenn-jocher** [comment]
@tetelevm official YOLOv5 inference methods and functions plot in ascending order so this is not an issue.

We do not provide support debugging custom code.

**Comment by glenn-jocher** [comment]
@tetelevm good news 😃! Your original issue may now be fixed ✅ in PR #3843. To receive this update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload with `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

Thank you for spotting this issue and informing us of the problem. Please let us know if this update resolves the issue for you, and feel free to inform us of any other **issues** you discover or **feature requests** that come to mind. Happy trainings with YOLOv5 🚀!
